discover/portal/articles/07-collaborating/01-publishing-blogs.markdown

```diff
@@ -11,9 +11,9 @@ Its editor delivers features you'd expect, in an elegant in-context manner. It
 has a complete set of WYSIWYG controls that appear where and when you need them.
 You can stay in this easy-to-use mode or switch to source mode to edit your
 content's HTML code. In source mode, you can work with light text on a dark
-background or dark text on a light background. To cap things off, you can do
-fullscreen editing to minimize distractions as your create your posts. You'll
-have a satisfying experience creating awesome blog posts. 
+background or dark text on a light background. To cap things off, you can
+enlarge the HTML editor window to minimize distractions as you create your
```

Comment:
Isn't the main benefit of this not the enlarging of the window, but the dual pane that shows the HTML on the left and the preview on the right?